### ISSUE TITLE
FreeBSD, "fcntl: Invalid argument"

### ISSUE THREAD

**Issue by filux** [MegaGlest/megaglest-source#72]
Message "fcntl: Invalid argument" is spamming console output multiple times, and first time shows very early during intro.

```
megaglest v3.12-dev
Compiled using: Clang: 3.4.1 [64bit] on: Oct  8 2015 12:56:29 platform: FreeBSD-X64 endianness: little
GIT: [Rev: 5170.02fe61e] - using STREFLOP [SSE] - [no-denormals]
fcntl: Invalid argument
fcntl: Invalid argument

**Comment by filux** [comment]
closed in waiting for changes related with #105

**Comment by filux** [comment]
Based on the https://github.com/MegaGlest/megaglest-source/issues/105#issuecomment-257027347, this issue is not reproducible on v3.12.0 (based on later commit than my tests were done), so issue will stay closed.
